README.md

```diff
@@ -67,6 +67,31 @@ sentry_dsn = "https://057006d7dfe5fff0fbed461cfca5f757@sentry.io/1111111"
 sentry_traces_sample_rate = 0.2  # 20% of requests will be logged under the performance tab
 ```
 
+`traces_sampler` can be used instead of `sentry_traces_sample_rate` to have a more granular control, [see details here](https://docs.sentry.io/platforms/rust/configuration/sampling/#configuring-the-transaction-sample-rate).
+```rust
+use rocket_sentry::RocketSentry;
+
+#[launch]
+fn rocket() -> _ {
+    let rocket_instance = rocket::build();
+    let default_rate = rocket_instance.figment().extract_inner::<f32>("sentry_traces_sample_rate").unwrap();
```

Comment:
I think the example in README should be as simple as possible. Let's drop the `default_rate` thing and merge `rocket::build()` + `.attach()` back together as one line.

You can instead mention that users can look at `examples/performance.rs` for more elaborate examples.